doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+MINHEIGHT(150) | E | 150 | Minimum pixel height for GUI dialog windows
+MINWIDTH(150) | E | 150 | Minimum pixel width for GUI dialog windows
+MNOTE(0) | MA | 0 | Control MNOTE error/warning messages during macro expansion and assembly: <br/> 0 - default generates MNOTE error/warning during mz390 and az390 <br/> 1 - suppress mz390 error/warning and only pass MNOTE to az390 <br/> 2 - generates MNOTE error/warning during mz390 and suppresses passing them on to az390
+MOD | L | NO | Generate raw code file from lz390 with .MOD suffix and no header or trailer and no RLD's. 
+OBJ | A | YES | Generate relocatable object code file
+OBJHEX | A | NO | Generate ASCII readable form of OBJ file with support for 31 bit long sections.  The default is to generate binary OBJ file which is compatible with mainframe linkers but is limited to 24 bit long sections.
+PARM('text') | E | none | Define character string parm to passed to executable program via address in R1 at startup pointing to a fullword pointing to a halfword length followed by EBCDIC characters. When the length is zero, no PARM is present.
+PC | M | YES | Generate macro pseudo code in cache memory for speeding up macro code that is executed more than once during macro expansion.
+PCOPT | M | YES | Optimize macro pseudo code by looking for push, add/sub, store sequences and replacing them with inc/dec pseudo code opcodes.  These pseudo codes are maintained in a cache in memory during macro execution only and have no affect on generated object code.
+PRINTALL | A | NO | Suppress PRINT OFF and PRINT NOGEN commands to force all source lines on PRN listing.
+PDSMEM8 | M | NO | If PDSMEM8 option is specified, error messages will be issued for any macro or copybook file names with length greater than 8.  This options is available to provide compatibility with mainframe PDS libraries that are limited to 8 character names. Note zCOBOL and zCICS use names longer than 8 characters such as zcobol\IDENTIFICATION.MAC and cics\CICS_INQUIRE.MAC.
+PROFILE(file) | M | NO | Insert COPY file in front of MLC source file
+PROLOG | M | YES | Generate CICS prolog macro calls DFHEIGBL, DFHEISTG, and  DFHEIENT if CICS option and PROLOG options are on.
+PROTECT | E | YES | Prevent modification of PSA in low memory 8k block
+REFORMAT | M | NO | Reformat expanded BAL code to align all opcodes at column 10 and operands at column 16 if possible.
+REGS | E | NO | Generate GPR register trace before and after each instruction if TRACE option on.
+RMODE24 | LE | YES | Set 390 load module options to load module below the 24 bit address line.
+RMODE31 | LE | NO | Set 390 load module options to load module above the 31 bit address line.  Note this options requires option MEM be set to include enough memory above the 31 bit address line.
```

Comment:
Line 85. Second sentence. "Note this options ..." -> "Note this option ...".

Comment: "options" in first sentence -- "... load module options ..." -- might be okay.

Reply:
Fixed 40c73aa